### PR TITLE
stop.sh: Kill cephadm deployed daemons

### PR DESCRIPTION
**Pull request by varshar16**
This patch adds call to cephadm vstart-cleanup script to remove daemons.

Signed-off-by: Varsha Rao <varao@redhat.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/stop.sh [hunk 1]
```diff
@@ -42,6 +43,10 @@ do_killall() {
     $SUDO killall -u $MYNAME $1
 }
 
+do_killcephadm() {
+    $SUDO $CEPHADM_DIR_PATH/vstart-cleanup.sh
```
Comment (sebastian-philipp) [inline src/stop.sh:47]: I don't think we need that. 

`cephadm rm-cluster --fsid $fsid --force`

should be simpler. 

Note that vstart supports restarting existing cluster, which is not what we're doing here. A similar operation would be something like `cephadm ls | jq .[].systemd_unit | xargs -n 1 systemctl disbale

**Comment by sebastian-philipp** [comment]
Getting;

```
➜  build git:(upstream-pull-35472) ✗ ../src/stop.sh ; finished                                                           
ERROR: cephadm should be run as root
ERROR: cephadm should be run as root
```

**Comment by varshar16** [comment]
> Getting
> 
> ```
> ➜  build git:(upstream-pull-35472) ✗ ../src/stop.sh ; finished                                                           
> ERROR: cephadm should be run as root
> ERROR: cephadm should be run as root
> ➜  cephadm git:(upstream-pull-35472) ✗ sudo ./cephadm ls
> [
>     {
>         "style": "cephadm:v1",
>         "name": "crash.ubuntu",
>         "fsid": "5f5a6f60-35dd-4ef2-b3e1-73a4ddc057a0",
>         "systemd_unit": "ceph-5f5a6f60-35dd-4ef2-b3e1-73a4ddc057a0@crash.ubuntu",
>         "enabled": true,
>         "state": "running",
>         "container_id": "51101f158c73a5c0281dba83fb1eb086551c9cf93ebc8c3d9dda14e3454fe041",
>         "container_image_name": "docker.io/ceph/daemon-base:latest-master-devel",
>         "container_image_id": "74803e884bea289d2d2d3ebdf6d37cd560499e955595695b1390a89800f4e37a",
>         "version": "16.0.0-901-g713ef3c",
>         "started": "2020-06-11T12:38:17.310159",
>         "created": "2020-06-11T12:38:17.117573",
>         "deployed": "2020-06-11T12:38:16.405570",
>         "configured": "2020-06-11T12:38:17.117573"
>     }
> ]
> ```

Fixed.

**Comment by varshar16** [comment]
> I think you're fixing https://tracker.ceph.com/issues/44746 here.

Yes, I have added the tracker ticket in the commit message.

File: src/stop.sh [hunk 2]
```diff
@@ -28,6 +28,7 @@ else
   CEPH_CONF_PATH="$PWD"
 fi
 conf_fn="$CEPH_CONF_PATH/ceph.conf"
+CEPHADM_DIR_PATH="$CEPH_CONF_PATH/../src/cephadm"
```
Comment (sebastian-philipp) [inline src/stop.sh:31]: This is going to make it even more complicated to have anything but one big cephadm script without any form of packaging.

**Pull request merged**